test/e2e/scenario/logs.scenario.ts

```diff
@@ -104,7 +101,7 @@ describe('logs', () => {
         // When reading the request, chunks length are probably not aligning perfectly with the
         // response length limit, so it sends few more bytes than necessary. Add a margin of error
         // to verify that it's still close to the expected limit.
-        DEFAULT_REQUEST_ERROR_RESPONSE_LENGTH_LIMIT * 2
+        DEFAULT_REQUEST_ERROR_RESPONSE_LENGTH_LIMIT * 4
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** this assertion is now pointless:

* `getLargeResponseWroteSize` is [around 100K](https://github.com/DataDog/browser-sdk/blob/0ab07cb70ac32cdd92677e80dd2942e7770e2cb9/test/e2e/lib/framework/serverApps/mock.ts#L7)
* `DEFAULT_REQUEST_ERROR_RESPONSE_LENGTH_LIMIT * 4` is [around 128K](https://github.com/DataDog/browser-sdk/blob/0ab07cb70ac32cdd92677e80dd2942e7770e2cb9/packages/logs/src/domain/configuration.ts#L35)

So this test does not ensure that we "read only the first bytes of the response" when doing a fetch.